tests/node-18.0/tests.js

```diff
@@ -1,37 +1,55 @@
 const fetch = require("node-fetch");
 
```

Comment:
This file makes more sense under `resources/functions`

Also let's rename `_resources` to `resources`

Reply:
Resources in this case is supposed to be in context of "resources for functions", not "resources for tests".

To make this clearer, we could have the `picture.png` in `tests/node-18.0/picture.png`, but that would make a lot of duplication of big files (repo will download for long time). That's why I prefered such a shared `resources` folder.

It was named `_resources` due to problems with gitignore file. Ill investigate that.

Reply:
Folder renamed from `_resources` to `resources`

Reply:
Well this is very similar to the case in Appwrite. We have tests ( which are the PHP files ) and tests have resources ( which can be functions, images etc etc. ) 

Here's what I'm suggesting

```
tests/
    - BaseV3.php
    - Node180.php
    - resources/
        - images/
            - picture.png
        - functions/
            - node18.js
            - node16.js     
```

Reply:
Updated. Actually, we no longer need `images` folder as this test and response was removed.